### DISCUSSION TITLE
Show group helpers' individual entities controls in the frontend

### DISCUSSION THREAD

**Discussion by DerangedWifi** [home-assistant/frontend#15192] [Dashboard]
**Display the controls/state of individual entities within a group under the group's info ui**

When showing any group that contains multiple entities, such as lights, locks, switches, binary sensors, etc, show each individual entity under the group's info ui as a collapsible card, much like how the current  automation builder works.

_Here's a rough markup I made on my phone. Could design something nicer looking later when on my pc._ 
![Screenshot_20230124-234207](https://user-images.githubusercontent.com/33496590/214508077-33acfef3-758a-4586-9801-5893b5d4eace.png)

This only makes it easier to understand what's going on in one place. With the additional option for device specific control within a group's existing UI, it makes sense to replace the currently empty unused space in favor of an easy to read, group entity summary/control stack of collapsible cards.

_Look at how Google home handles this - with some markup, that is._
![Screenshot_20230125-000042](https://user-images.githubusercontent.com/33496590/214510493-b220a0d4-ba98-4f63-8907-88ddb7edb4b6.png)

_Or even how it's been set up in my own Lovelace dash - the group controller for inputs meant to be sent to all devices in the group, with individual devices of the group listed below with control for each specifically._
![Screenshot_20230125-000607](https://user-images.githubusercontent.com/33496590/214510895-0880e8cc-69c6-463e-8459-037a02379b18.png)

What do you think?

**Comment by Wesley-Vos** [comment]
+1 for this feature request!

It would be really helpful if the states of the entities of a group (created by the helper UI) are shown in the more-info pop-up. It makes it easier to control the individual entities.

Imagine you have a light group with all lights from the living room which you almost always switch together. However, at some moment in time, you want to turn a specific light on or of. In stead of having cards for all lights on your dashboard or search for the individual light, it is way easier to open the more-info pop-up of the group (for example with a double tap action on the living room light button card) and turn the specific light on or off.

Hope it makes sense! Furthermore, I still have the idea that this was always the case for the old, YAML-generated groups, but not sure.

**Comment by dolfelt** [comment]
I have begun attempting to implement this feature. I will let you know when I have something to show for it @DerangedWifi and @Wesley-Vos

**Reply by dolfelt** [reply]
Initial PR (https://github.com/home-assistant/frontend/pull/25687) has been submitted with some screenshots.
